build.sbt

```diff
@@ -2,14 +2,21 @@ name := "Conseil"
 scalaVersion := "2.12.8"
 
 val akkaHttpVersion = "10.1.0"
-val akkaVersion = "2.5.11"
+val akkaVersion = "2.5.19"
```

Comment:
no breaking change? semantic versioning seems to work for real...

Reply:
from what I've tested it works just fine, I had some warnings about incompatibilities between akka libraries, that's why I introduced newer version.

Reply:
Good, I actually did a through job of finding incompatibilities and will probably add something more to the updates later on!